src/features/room/list/ui/HomeRoomStage.module.css

```
@@ -0,0 +1,143 @@
+  position: absolute;
+  left: var(--slot-left, 50%);
+  top: 50%;
+  transform: translate(-50%, -50%);
+  transition: left 420ms cubic-bezier(0.22, 1, 0.36, 1);
+  will-change: left;
+}
+
+.slotCard {
+  --card-width: var(--card-side-size);
+  --card-height: var(--card-side-size);
+  transform-origin: center center;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)